llm/README.md

```diff
@@ -0,0 +1,71 @@
+# Personal CHTC Submit Template for LLM Fine-Tuning
+
+Use Case: Fine-tuning large language models on CHTC and track it with `Weights & Biases`.
+
+![WANDB](wandb.png)
+
+## Quick start
+
+1. Store your WANDB credentials and `STAGING_DIR` path in an environment file named .env. See the provided [example](.env.example).
+1. Update the `run_name` in `run.sh`. This will be utilized as the WANDB tracking ID, and checkpoints will be saved in `STAGING_DIR/run_name/`.
+1. Modify `run.sub` as necessary.
+1. SSH to the submit node.
+1. Create a `condor_log` directory using the command: `mkdir condor_log` if you don't have it.
+1. Submit your job using `condor_submit run.sub`.
+
+## Used Stacks
+
+- Docker
+- Github Container Registry (ghcr.io)
+- Huggingface Transformers
+- Weights & Biases (WANDB)
+
+## Used CHTC/HTCondor Features
+
+- Docker Universe
+- Checkpointing
+- Staging (for storing checkpoints)
```

Comment:
@ChristinaLK this template requires staging access. How should we document instructions for requesting that? Or do you have ideas to store checkpoints in a different way?

Reply:
let's leave it for now...